shakedown/dcos/service.py

```diff
@@ -225,6 +252,28 @@ def wait_for_mesos_task_removal(task_name, timeout_sec=120):
     return time_wait(lambda: mesos_task_not_present_predicate(task_name), timeout_seconds=timeout_sec)
 
 
+def delete_persistent_data(role, principal, zk_node):
```

Comment:
@nickbp totally makes sense that this is temp and this code is doing what we document our customers to do... however if I remember correctly janitor is python code in the docker... we would much prefer to not introduce docker dependencies into shakedown.   The right way to do this is pull the python code from janitor into shakedown.

I chatted with @sschneid.  We had a JIRA sometime ago to do this.    Please pull this code from the PR so we can merge the majority of this (which is great stuff!).   @sschneid indicated that he would create a new PR pulling the janitor code into shakedown ( I assume using your API).